templates/alertmanager/alertmanager-statefulset.yaml

```
@@ -175,6 +175,15 @@ spec:
             - name: gossip
               containerPort: {{ .Values.config.memberlist.bind_port }}
               protocol: TCP
+            - name: grpc
+              containerPort: {{ .Values.config.server.grpc_listen_port }}
+              protocol: TCP
+            - containerPort: 9094
```

I'm conflicted on this. The magic number `9094` bothers me. It would be nice to be able to support the configuration and only default to `9094`, but that's awkward because the port number is embedded at the end of the listen string. Something like this (but this is kind of ugly):
```suggestion
            - containerPort: {{ (.Values.config.alertmanager.cluster.listen_address | default "0.0.0.0:9094" | split ":").1 }}
```

---

> It would be nice to be able to support the configuration
Would you mind to point me where we can tweak the port in the configuration in regards to gossip cluster ? I think it's a good idea to tweak the magic number 9094 but I really don't know where we can tell the alertmanager in the first place to use a different port when running in cluster mode for gossip.

---

What about introducing an extra variable with a docstring in `values.yaml`?

---

+1 on extra var but how do we make alertmanager container ack a new variable from values.yaml ? I understand currently the way to change 9094 is within the `.Values.config.alertmanager.cluster.listen_address`